src/azure-cli/azure/cli/command_modules/appconfig/commands.py

```diff
@@ -61,3 +75,20 @@ def load_command_table(self, _):
         g.command('restore', 'restore_key')
         g.command('import', 'import_config')
         g.command('export', 'export_config')
+
+    # FeatureManagement Commands
+    with self.command_group('appconfig feature', configstore_feature_util) as g:
+        g.command('set', 'set_feature')
+        g.command('delete', 'delete_feature')
+        g.command('show', 'show_feature')
+        g.command('list', 'list_feature')
+        g.command('lock', 'lock_feature')
+        g.command('unlock', 'unlock_feature')
+        g.command('enable', 'enable_feature')
+        g.command('disable', 'disable_feature')
```

Comment:
It looks all your commands are custom_command. Can you change it to custom_command?